BuildResidentialHPXML/measure.rb

```diff
@@ -2635,6 +2636,306 @@ def arguments(model) # rubocop:disable Lint/UnusedMethodArgument
     arg.setDefaultValue(4000)
     args << arg
 
+    arg = OpenStudio::Measure::OSArgument.makeStringArgument('electric_panel_service_feeders_load_calculation_types', false)
+    arg.setDisplayName('Electric Panel: Service/Feeders Load Calculation Types')
+    arg.setDescription("Types of electric panel service/feeder load calculations. Possible types are: #{HPXML::ElectricPanelLoadCalculationType2023ExistingDwellingLoadBased}, #{HPXML::ElectricPanelLoadCalculationType2023ExistingDwellingMeterBased}. If multiple types, use a comma-separated list. If not provided, no electric panel loads are calculated.")
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('electric_panel_baseline_peak_electricity_power', false)
+    arg.setDisplayName('Electric Panel: Peak Electricity Power')
+    arg.setDescription("Specifies the baseline peak electricity. Used for #{HPXML::ElectricPanelLoadCalculationType2023ExistingDwellingMeterBased}.")
+    arg.setUnits('W')
+    args << arg
+
+    electric_panel_voltage_choices = OpenStudio::StringVector.new
+    electric_panel_voltage_choices << HPXML::ElectricPanelVoltage120
+    electric_panel_voltage_choices << HPXML::ElectricPanelVoltage240
+
+    arg = OpenStudio::Measure::OSArgument::makeChoiceArgument('electric_panel_service_voltage', electric_panel_voltage_choices, false)
+    arg.setDisplayName('Electric Panel: Service Voltage')
+    arg.setDescription("The service voltage of the electric panel. If not provided, the OS-HPXML default (see <a href='#{docs_base_url}#hpxml-electric-panels'>HPXML Electric Panels</a>) is used.")
+    arg.setUnits('V')
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('electric_panel_service_rating', false)
+    arg.setDisplayName('Electric Panel: Service Rating')
+    arg.setDescription("The service rating of the electric panel. If not provided, the OS-HPXML default (see <a href='#{docs_base_url}#hpxml-electric-panels'>HPXML Electric Panels</a>) is used.")
+    arg.setUnits('A')
+    args << arg
+
+    electric_panel_breaker_spaces_type_choices = OpenStudio::StringVector.new
+    electric_panel_breaker_spaces_type_choices << 'total'
+    electric_panel_breaker_spaces_type_choices << 'headroom'
+
+    arg = OpenStudio::Measure::OSArgument::makeChoiceArgument('electric_panel_breaker_spaces_type', electric_panel_breaker_spaces_type_choices, false)
+    arg.setDisplayName('Electric Panel: Breaker Spaces Type')
+    arg.setDescription("The breaker spaces specification type of the electric panel. If not provided, the OS-HPXML default (see <a href='#{docs_base_url}#hpxml-electric-panels'>HPXML Electric Panels</a>) is used.")
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeIntegerArgument('electric_panel_breaker_spaces', false)
+    arg.setDisplayName('Electric Panel: Breaker Spaces')
```

Comment:
I asked this earlier, but can we allow both inputs? Then we can have two simple arguments (one for total, one for headroom) and not need the user to specify a type argument that defines the second argument.